src/client/src/app/components/merch/merch.component.ts

```diff
@@ -25,4 +27,9 @@ export class MerchComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  navigateToDetails() {
+    console.log("clicked");
```

Comment:
remember to try to leave console logs out of pull request code.

Reply:
good catch! I ll remove that

Reply:
ok and i guess you would resolve this after your change it??